shopinvader/models/shopinvader_partner.py

```diff
@@ -90,14 +90,24 @@ def _is_same_partner_value(self, partner, vals):
         """ we check if one of the given value is different than values
             of the given partner
         """
-        data = partner._convert_to_write(partner._cache)
-        for key in data:
-            if key not in vals:
+        skip_keys = self._is_same_partner_value_skip_keys(partner)
+        keys_to_check = []
+        for key in vals.keys():
+            if key in skip_keys or key not in partner:
                 continue
+            keys_to_check.append(key)
+            if key not in partner._cache:
+                partner[key]  # load missing key
```

Comment:
```suggestion
                # pylint: disable=pointless-statement
                partner[key]  # load missing key
```
@simahawk IMO pylint is failing due to this line https://travis-ci.org/shopinvader/odoo-shopinvader/jobs/611453281#L355
![image](https://user-images.githubusercontent.com/544090/68875291-e1551500-0702-11ea-8081-bd6b770570a0.png)